server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -0,0 +1,80 @@
+package services.ti;
+
+import com.google.common.base.Strings;
+import forms.AddApplicantToTrustedIntermediaryGroupForm;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import javax.inject.Inject;
+import models.TrustedIntermediaryGroup;
+import play.data.Form;
+import repository.UserRepository;
+import services.DateConverter;
+
+public class TrustedIntermediaryService {
+  private final UserRepository userRepository;
+  private final DateConverter dateConverter;
+
+  @Inject
+  public TrustedIntermediaryService(UserRepository userRepository, DateConverter dateConverter) {
+    this.userRepository = userRepository;
+    this.dateConverter = dateConverter;
+  }
+
+  public TIClientCreationResult addNewClient(
+      Form<AddApplicantToTrustedIntermediaryGroupForm> form,
+      TrustedIntermediaryGroup trustedIntermediaryGroup)
+      throws EmailAddressExistsException {
+    form = validateEmailAddress(form);
+    form = validateFirstName(form);
+    form = validateLastName(form);
+    form = validateDateOfBirth(form);
+    if (form.hasErrors()) {
+      return TIClientCreationResult.failure(form);
+    }
+    userRepository.createNewApplicantForTrustedIntermediaryGroup(
+        form.get(), trustedIntermediaryGroup);
```

Comment:
thought (non-blocking): it's outside of the scope of this PR, but a good code improvement for the future would be to move the logic in `UserRepository` that's related to TIs into this new service you've created. That would fit the intention of keeping repositories focused on query logic only.

Reply:
I agree with you! we should move a bit of code around and address it in the subsequent PRs.